### DISCUSSION TITLE
Kubernetes integration

### DISCUSSION THREAD

**Discussion by maitredede** [modoboa/modoboa#2150] [Help]
Hello,

I am looking for mail stack solutions to replace our hand-made one (dovecot, postfix, spamassassin, opendkim, ldap, no gui, only a webmail) with one easy to configure and use, preferably with a GUI, that can be hosted on kubernetes. And I found modoboa.

It seams that this project fits my needs, but I would like to know if there is a working version of modoboa working on docker, with components splitted on different containers (a docker-compose file would make it).

Where can I start ? I will provide feedback of mail stack migration.

Thanks.

**Comment by tonioo** [comment]
There is not official docker build for Modoboa but you can find some community made images (example: https://hub.docker.com/r/ntninja/modoboa).
